### PR TITLE
feat: added descriptions to `pypi` and fixed some typos

### PR DESCRIPTION
**Pull request by Arfey**
I have problem with the picture, but I think it's not so important now.

screen shot 2018-10-08 at 00 53 51

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -60,17 +68,21 @@ def read_version():
...
     version=read_version(),
-    description=("The tool that helps quickly create a basis for "
-                 "the microservice on aiohttp and prepare the development "
-                 "environment."),
+    description=(
+        "The tool that helps quickly create a basis for "
+        "the microservice on aiohttp and prepare the development "
+        "environment."
+    ),
+    long_description=get_text_from('README.md'),
```
Comment (webknjaz) [inline setup.py]: If you switch to rst, you'll be able to embed (parts of) readme into sphinx docs

**Comment by webknjaz** [comment]
Picture problem is there because the url is relative to github and there's none @ pypi (you cannot upload such artifacts there)

File: create_aio_app/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-__version__ = '0.0.1'
+__version__ = '0.0.2'
```
[the remark this answers is not in the document]
Comment (Arfey) [reply create_aio_app/__init__.py]: it's temporary

File: setup.py (hunk 1 above)
Comment (Arfey) [reply setup.py]: \+

File: setup.py [hunk 3]
```diff
@@ -60,17 +68,21 @@ def read_version():
...
     name="create-aio-app",
     version=read_version(),
-    description=("The tool that helps quickly create a basis for "
-                 "the microservice on aiohttp and prepare the development "
-                 "environment."),
+    description=(
+        "The tool that helps quickly create a basis for "
+        "the microservice on aiohttp and prepare the development "
+        "environment."
+    ),
```
[the remark this answers is not in the document]
Comment (Arfey) [reply setup.py]: I prefer use brackets

**Comment by Arfey** [comment]
Yes, u right. I rewrite this to absolute url.

**Pull request merged**